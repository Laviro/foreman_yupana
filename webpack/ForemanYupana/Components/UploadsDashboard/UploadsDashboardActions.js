import { UPLOADSDASHBOARD_CHANGE_BOOL } from './UploadsDashboardConstants';

export const changeBool = resource => dispatch => {
  dispatch({
    type: UPLOADSDASHBOARD_CHANGE_BOOL,
    payload: resource,
  });
};
