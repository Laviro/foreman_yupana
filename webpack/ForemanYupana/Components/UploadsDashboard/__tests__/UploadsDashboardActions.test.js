import { testActionSnapshotWithFixtures } from 'react-redux-test-utils';
import { changeBool } from '../UploadsDashboardActions';

const fixtures = {
  'should changeBool': () => changeBool({ bool: true }),
};

describe('UploadsDashboard actions', () =>
  testActionSnapshotWithFixtures(fixtures));
