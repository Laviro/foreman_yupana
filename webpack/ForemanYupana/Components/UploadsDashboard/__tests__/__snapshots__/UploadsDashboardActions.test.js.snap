// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UploadsDashboard actions should changeBool 1`] = `
Array [
  Array [
    Object {
      "payload": Object {
        "bool": true,
      },
      "type": "UPLOADSDASHBOARD_CHANGE_BOOL",
    },
  ],
]
`;
